client/my-sites/site-settings/delete-site/index.jsx

```diff
@@ -134,7 +135,7 @@ class DeleteSite extends Component {
 						onChange={ this.onConfirmDomainChange }
 						value={ this.state.confirmDomain }
 						aria-required="true"
-						id="confirmDomainChangeInput"
+						data-testid="confirmDomainChangeInput"
```

Comment:
Can we access this element in test without using a testid?

Reply:
Yes, I just updated it; thanks for your review!